autoware_adapi_v1_msgs/perception/msg/Shape.msg

```diff
@@ -0,0 +1,8 @@
+uint8 UNKNOWN=0
+uint8 BOUNDING_BOX=1
```

Comment:
```suggestion
uint8 BOX=1
```

Reply:
Same question with `POLYGON`

Reply:
I referred to [shape_msgs](https://github.com/ros2/common_interfaces/blob/rolling/shape_msgs/msg/SolidPrimitive.msg). Either one is fine.

Reply:
changed at 
f6bfa5e